src/tests/createHeadlessForm.test.js

```diff
@@ -820,6 +821,45 @@ describe('createHeadlessForm', () => {
       });
     });
 
+    // @BUG COD-1859
+    // it should validate when type is string but value is not a boolean
+    it('support "radio" field string-y type', () => {
```

Comment:
**Suggestion** 

Not sure this needs a separate test case.
We could just ensure the string test case on line 744 ensures a string by updating it to use:
```
          options: [
            {
              label: 'Yes',
              value: 'true',
            },
            {
              label: 'No',
              value: 'false',
            },
          ]
```
Then we can just add this there:
```
      expect(validateForm({ has_siblings: false })).toEqual({
        has_siblings: 'The option "false" is not valid.',
      });
```